src/app/researcher/study/[encodedStudyId]/review/runs-table.tsx

```diff
@@ -79,17 +79,21 @@ const RunsTable: React.FC<RunsTableProps> = ({ isActive, study }) => {
                             <Table.Td>{run.startedAt?.toISOString() || ''}</Table.Td>
                             <Table.Td align="right">
                                 <Group>
-                                    {run.status == 'INITIATED' && (
-                                        <>
-                                            <Modal opened={opened} onClose={close} title="AWS Instructions" centered>
-                                                <PushInstructions
-                                                    containerLocation={study.containerLocation}
-                                                    runId={run.id}
-                                                />
-                                            </Modal>
-                                            <Button onClick={open}>View Instructions</Button>
-                                        </>
-                                    )}
+                                    <>
+                                        <Modal
+                                            size={800}
+                                            opened={opened}
+                                            onClose={close}
+                                            title="AWS Instructions"
+                                            centered
+                                        >
+                                            <PushInstructions
+                                                containerLocation={study.containerLocation}
+                                                runId={run.id}
+                                            />
+                                        </Modal>
+                                        <Button onClick={open}>View Instructions</Button>
+                                    </>
                                     {run.status == 'COMPLETED' && <PreviewCSVResultsBtn run={run} />}
```

Comment:
we do not want to show the instructions after they've already uploaded their code, so we need to keep the `{run.status == 'INITIATED' && (`